pyclang/runner.py

```diff
@@ -25,13 +26,27 @@ def _is_exe(filepath: str) -> bool:
     return False
 
 
+# Modify PATHEXT only once
+@lru_cache(maxsize=None)
```

Comment:
This is the same as `@cache()` if `maxsize=None`, I think.

Reply:
> Returns the same as lru_cache(maxsize=None), creating a thin wrapper around a dictionary lookup for the function arguments. Because it never needs to evict old values, this is smaller and faster than [lru_cache()](https://docs.python.org/3/library/functools.html#functools.lru_cache) with a size limit.

https://docs.python.org/3/library/functools.html#functools.cache

But it needs Python 3.9. So that might be a limiting factor.